src/main/java/net/sf/rails/game/financial/ShareSellingRound.java

```diff
@@ -25,7 +25,7 @@
 
 // TODO: Check if un-initialized states cause undo problems
 public class ShareSellingRound extends StockRound {
-    private static final Logger log = LoggerFactory.getLogger(ShareSellingRound.class);
+    protected static final Logger log = LoggerFactory.getLogger(ShareSellingRound.class);
```

Comment:
Should be private

Reply:
Cant be, inherited by a class that needs access to that logger...